### DISCUSSION TITLE
definitionHover: how to increase width?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#4259] [Q&A] [answered]
Currently not enough:
![image](https://user-images.githubusercontent.com/3514015/193881879-59a0435b-0bd2-4904-a2f7-67dcc0963ddd.png)

I tried 

`call coc#config('diagnostic', { 'floatConfig': {'maxWidth': 120 }})`

**Comment by chemzqm** [comment, marked as answer]
Seems you need `call coc#config('hover', { 'floatConfig': {'maxWidth': 120 }})` or `call coc#config('floatFactory', { 'floatConfig': {'maxWidth': 120 }})`
